cm-gateway/gateway-api-install/main.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "~> 2.12.1"
+    }
+  }
+}
+
+provider "kubernetes" {
+  config_path = pathexpand(var.kubeconfig_path)
+}
+
+locals {
+  # For windows OS, it need to use "---\r\n"
+  raw_gateway_manifests = split("---\n", file("./manifests/gateway-api.yaml"))
```

Comment:
There is a difference between Windows and Unix.

Reply:
Makes sense, I don't have a Windows machine available to test with